web/src/app/modules/data/modules/data-service/services/specmate-data.service.ts

```diff
@@ -240,8 +238,7 @@ export class SpecmateDataService extends Monitorable {
             this.end(SpecmateDataService.OP_COMMIT);
             this.committed.emit();
         } catch (error) {
-            this.simpleModal.openOk(this.translate.instant('saveError.title'), this.translate.instant('saveError.retry'));
```

Comment:
I think a warning to the user would be useful. Otherwise the changes get lost and the user does not even know it.